config/default.yaml

```diff
@@ -300,6 +300,20 @@ ecosystem:
     max_size_mb: 0
     max_go_threads: 10000
 
+# environment release information
+release:
+  # release mode stable or canary
+  mode: stable
+  # name of cookie
+  cookieName: biscotti
```

Comment:
let's call these something related to SD ?
`cookieName:release`, `cookieValue: stable`, header should follow suit